verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -313,17 +313,26 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                             pageSizeOptions: [5, 10, 20, 50, 100],
                         }}
                         editable={{
-                            onRowAdd: (
-                                newRowData: TableRow,
-                            ): Promise<unknown> => this.addCase(newRowData),
-                            onRowUpdate: (
-                                newRowData: TableRow,
-                                oldRowData: TableRow | undefined,
-                            ): Promise<unknown> =>
-                                this.editCase(newRowData, oldRowData),
-                            onRowDelete: (
-                                rowData: TableRow,
-                            ): Promise<unknown> => this.deleteCase(rowData),
+                            onRowAdd:
+                                this.props.user.roles.indexOf('curator') !== -1
```

Comment:
`roles.includes('curator') ? ... : ...` is a little more clear IMO

Reply:
thanks, done